### PR TITLE
[BUG] New Item from Multiple RSS Feeds not working

### LINKED ISSUE #4165: [BUG] New Item from Multiple RSS Feeds not working
**Describe the bug**
Having issues with Pipedream reliably triggering when I am tracking multiple RSS feeds. Often times no new triggers will happen even as I know the RSS feeds are being updated with new items.

**To Reproduce**
1. Go to Pipedream and set up the "New Item from Multiple RSS Feeds" Trigger
2. Add multiple feeds (ideally 8-10)
3. Set up the feed to trigger on a recurring basis (e.g. daily)
4. Monitor events and notice nothing new is coming in

**Expected behavior**
When I only had 2 feeds on the trigger (https://oversharing.substack.com/feed.xml and https://noahpinion.substack.com/feed.xml), everything triggered on a daily basis as expected without any problems. But then I added multiple more feeds (see below) and then the events stopped coming through, nothing was in the logs, etc. 

Feeds:
https://mattstoller.substack.com/feed.xml
https://modernpower.substack.com/feed.xml
https://cityobservatory.org/feed/
https://www.apricitas.io/feed.xml
https://herbsundays.substack.com/feed.xml
https://erikhoel.substack.com/feed.xml
https://theconnector.substack.com/feed.xml
https://alltechishuman.substack.com/feed.xml

**Screenshots**
Before the 30th of August, I only had 2 feeds, and then I updated to multiple more feeds (see above). As you can see in the logs, something weird starting happening once I added the additional feeds, and no events were available for me to investigate.
image

**Desktop:**
 - OS: Mac OS
 - Browser: Chrome
 - Version 104

**Additional context**
I enabled the option for Pipedream Support to access/update my workflow: https://pipedream.com/@nkav/fewer-rss-feeds-transform-to-markdown-in-todoist-p_mkC9JAl/build

### PR DESCRIPTION
**Pull request by michelle0927**
Resolves #4165 

I'm not sure why we don't get a timeout error, but the source appears to be timing out when trying to handle too much data. To help avoid this, I added a `max` prop (maximum posts per feed to return at one time). Setting it at 10, I was able to successfully run the source with 7 URLs. I also updated the description of the `urls` prop to recommend entering 5 or less URLs.

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/4204"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible
- [ ] [Create subscription webhooks sources (vs polling sources) whenever possible](https://pipedream.com/docs/components/guidelines/#webhook-vs-polling-sources)
- [ ] [Always emit a `summary` with events](https://pipedream.com/docs/components/guidelines/#emit-a-summary)
- [ ] [Use a built-in deduping strategy wherever possible](https://pipedream.com/docs/components/guidelines/#deduping)
- [ ] When building webhook sources, [follow these best practices](https://pipedream.com/docs/components/guidelines/#webhook-sources)
- [ ] When building polling sources, [follow these best practices](https://pipedream.com/docs/components/guidelines/#polling-sources)

File: components/rss/sources/new-item-from-multiple-feeds/new-item-from-multiple-feeds.ts [hunk 1]
```diff
@@ -5,17 +5,25 @@ import rssCommon from "../common/common";
...
       propDefinition: [
         rss,
         "urls",
       ],
+      description: "Enter one or multiple URLs from any public RSS feed. To avoid timeouts, 5 or less URLs is recommended.",
+    },
+    max: {
+      type: "integer",
+      label: "Max per Feed",
+      description: "Maximum number of posts per feed to retrieve at one time",
```
Comment (andrewjschuang) [inline components/rss/sources/new-item-from-multiple-feeds/new-item-from-multiple-feeds.ts:24]: I think we should add the default value here so the user knows how much it is

**Review by andrewjschuang — COMMENTED** [review]
@michelle0927, thanks for the contribution!

I would suggest adding some logs so the user knows what is the progress. Though it seems like the source will only log everything if it concludes successfully.

@dannyroosevelt @dylburger is this the case? Shouldn't the source log while it is executing?

Btw, moving to QA since this doesn't affect the results.

**Review by andrewjschuang — APPROVED** [review]

**Pull request merged**